app/celery/tasks.py

```diff
@@ -269,3 +270,49 @@ def recreate_pdf_for_precompiled_letter(notification_id, file_location, allow_in
             notification_id,
         )
         return
+
+
+@notify_celery.task(name="recreate-pdf-for-template-letter-attachments")
+def recreate_pdf_for_template_letter_attachments(service_id, attachment_id, original_filename):
```

Comment:
To confirm if I understand correctly: we add this task, so if this deploy fails and we have to roll back and re-process some letters, we can re-sanitise the letter attachments?

In that case, shouldn't this new task be in a separate PR, deployed before the Trixie update? So that we can still access it if we roll back Trixie update PR?